Common/Securities/Volatility/IVolatilityModel.cs

```diff
@@ -34,6 +35,14 @@ public interface IVolatilityModel
         /// <param name="security">The security to calculate volatility for</param>
         /// <param name="data">The new data used to update the model</param>
         void Update(Security security, BaseData data);
+
+        /// <summary>
+        /// Returns history requirements for the volatility model expressed in the form of history request
+        /// </summary>
+        /// <param name="security">The security of the request</param>
+        /// <param name="utcTime">The date/time of the request</param>
+        /// <returns>History request object, or null if no requirements</returns>
+        HistoryRequest HistoryRequirements(Security security, DateTime utcTime);
```

Comment:
Awesome!!! Allowing each vol model to define their own historical requirements is very smart and good use of OO principles. 

Perhaps for later extensibility this should return `IEnumerable<HistoryRequest>`, as it's not unreasonable that a vol model may look at the last 100 daily bars for a longer term view and perhaps the last week of hourly data for a shorter term view. Also recommend renaming the method. Per naming guidelines, methods should be verbs, but this is a noun.

Is this method only invoked to seed the model or is it invoked more regularly? If it's only invoked once at startup (which it appears to be at the top of `AlgorithmManager.Stream` then perhaps a name that alludes to the fact that these `HistoryRequest` instance(s) are going to be pumped into the model during algorithm startup/warmup phase.